### PR TITLE
[EHN] OTX Feed pulse_key

### PR DESCRIPTION
**Pull request by navtej**
OTX Feed recently introduced `pulse_key` in their pulse indicators. This key helps in aggregating all IoCs for a given pulse.

This should be merged after #1009

### CODE REVIEW

File: intelmq/tests/bots/parsers/alienvault/test_parser_otx.py [hunk 1]
```diff
@@ -46,28 +47,74 @@
...
   'comment': 'Active users of mobile banking apps should be aware of a new '
-             'Android banking malware campaign targeting customers of large '
-             'banks in the United States, Germany, France, Australia, Turkey, '
-             'Poland, and Austria. This banking malware can steal login '
-             'credentials from 94 different mobile banking apps. Due to its '
-             'ability to intercept SMS communications, the malware is also '
-             'able to bypass SMS-based two-factor authentication. '
-             'Additionally, it also contains modules to target some popular '
-             'social media apps.',
+                 'Android banking malware campaign targeting customers of large '
```
Comment (wagner-certat) [inline intelmq/tests/bots/parsers/alienvault/test_parser_otx.py]: Why are you indenting here?

File: intelmq/tests/bots/parsers/alienvault/test_parser_otx.py [hunk 2]
```diff
@@ -46,28 +47,74 @@
...
   'malware.hash.sha256': 'e5df30b41b0c50594c2b77c1d5d6916a9ce925f792c563f692426c2d50aa2524',
   'raw': 'eyJhY2Nlc3NfZ3JvdXBzIjogW10sICJhY2Nlc3NfcmVhc29uIjogIiIsICJhY2Nlc3NfdHlwZSI6ICJwdWJsaWMiLCAiY29udGVudCI6ICIiLCAiY3JlYXRlZCI6ICIyMDE2LTExLTAzVDIwOjE1OjQ0IiwgImRlc2NyaXB0aW9uIjogIiIsICJleHBpcmF0aW9uIjogbnVsbCwgImlkIjogMTI2NTM1NCwgImluZGljYXRvciI6ICJlNWRmMzBiNDFiMGM1MDU5NGMyYjc3YzFkNWQ2OTE2YTljZTkyNWY3OTJjNTYzZjY5MjQyNmMyZDUwYWEyNTI0IiwgImlzX2FjdGl2ZSI6IDEsICJvYnNlcnZhdGlvbnMiOiAzLCAicm9sZSI6IG51bGwsICJ0aXRsZSI6ICIiLCAidHlwZSI6ICJGaWxlSGFzaC1TSEEyNTYifQ==',
   'time.source': '2016-11-03T20:01:00+00:00'
   }
 
+EXAMPLE_EVENT_3 = {
+  '__type': 'Event',
+   'classification.type': 'blacklist',
+   'comment': 'Myanmar is a country currently engaged in an important '
+                 'political process. A pro-democracy reform took place \n'
```
Comment (wagner-certat) [inline intelmq/tests/bots/parsers/alienvault/test_parser_otx.py]: same here: strange indent.

**Comment by wagner-certat** [comment]
@robcza or @e3rd can you have a look please?

File: intelmq/tests/bots/parsers/alienvault/test_parser_otx.py (hunk 2 above)
Comment (navtej) [inline intelmq/tests/bots/parsers/alienvault/test_parser_otx.py]: different editors on diff machines.

**Comment by navtej** [comment]
Not very sure about sharing data in .data file. Fundamentally it is not different from what was already there, so i assumed if it was ok to share that data, it should be ok. Particularly Hidden Cobra pulse was chosen because, indicators for that, came from a very public source ie US-CERT. However, it would be wiser to talk to alienvault about it, as well ask them who owns the rights for data in case the pulse is authored by someone other than AlienVault.

**Comment by wagner-certat** [comment]
If you are allowed to share the data I have no problem. Just want to make sure it is the case :)

**Comment by e3rd** [comment]
(It seems great, thanks navtej )

**Pull request merged**